src/components/calcite-action/calcite-action.scss

```diff
@@ -64,16 +64,31 @@
   }
 }
 
-:host([scale="s"]) .button {
-  @apply p-2;
+:host([scale="s"]) {
+  .button {
+    @apply p-2 text--2h;
+  }
+  .button--text-visible .icon-container {
+    @apply mr-2;
+  }
```

Comment:
If this is set here, we should probably also remove margin styles on `icon-container` I think around line 124.

Reply:
Sounds good. I will leave only keep `m-0` on line 124.

Reply:
@asangma Updated.

Reply:
🧹